apps/theming-designer/src/components/Header.tsx

```diff
@@ -25,22 +36,33 @@ const outputPanelClassName = mergeStyles({
 
 const textAreaClassName = mergeStyles({
   height: 350,
-  width: 280,
+  width: '100%',
   marginRight: 28,
   backgroundColor: 'white',
   color: '#333'
 });
 
 const microsoftLogo = mergeStyles({
```

Comment:
Observation: Direct use of mergeStyles in the code feels bad, especially when you have "styles" props which aren't.. It creates too much ambiguity around when and when not to use.

Each of these components should be married to styling with the factory function.

```
const Logo = styled('div', {
  styles: () => { ... IStyles... }
});
```

Reply:
Yeah, i wouldn't mind cleaning all of those up eventually. Really focusing on the proper way to create a 'styledComponent' would help here.